### ISSUE TITLE
Getting validation error when requesting funding history

### ISSUE THREAD

**Issue by joyceyuu** [stabacco/stake-python#155]
```
pydantic.error_wrappers.ValidationError: 8 validation errors for Fundings
items -> 2 -> action
  value is not a valid enumeration member; permitted: 'DEPOSIT', 'DIVIDEND_DEPOSIT', 'SETTLEMENT', 'TRANSFER', 'WITHDRAWAL' (type=type_error.enum; enum_values=[<Action.DEPOSIT: 'DEPOSIT'>, <Action.DIVIDEND_DEPOSIT: 'DIVIDEND_DEPOSIT'>, <Action.SETTLEMENT: 'SETTLEMENT'>, <Action.TRANSFER: 'TRANSFER'>, <Action.WITHDRAWAL: 'WITHDRAWAL'>])
items -> 2 -> approvedBy
  value is not None (type=type_error.not_none)
items -> 4 -> action
  value is not a valid enumeration member; permitted: 'DEPOSIT', 'DIVIDEND_DEPOSIT', 'SETTLEMENT', 'TRANSFER', 'WITHDRAWAL' (type=type_error.enum; enum_values=[<Action.DEPOSIT: 'DEPOSIT'>, <Action.DIVIDEND_DEPOSIT: 'DIVIDEND_DEPOSIT'>, <Action.SETTLEMENT: 'SETTLEMENT'>, <Action.TRANSFER: 'TRANSFER'>, <Action.WITHDRAWAL: 'WITHDRAWAL'>])
items -> 4 -> approvedBy
  value is not None (type=type_error.not_none)
```

**Comment by joyceyuu** [comment]
I've made a branch to fix it but unfortunately I don't have permisson to push it to the remote. Here are the specifix fixes:
1. There is a new type of action "ADJUSTMENT" in stake/asx/funding.py
2. The argument approveBy is not None for some transactions but it was set to be None
3. total_items in Fundings has an alias "totalItems"

Check out the diff below for details. Those changes fixed my issue
<img width="671" alt="Screenshot 2023-02-02 at 8 31 34 pm" src="https://user-images.githubusercontent.com/10160933/216285986-25f18309-3323-45d1-b4ad-f3b57dcf7a87.png">

**Comment by stabacco** [comment]
Can you try forking the repo and submitting a PR from there?

**Comment by stabacco** [comment]
Should be fixed in 0.6.4. Closing, please re-open if you still have problems.
